### DISCUSSION TITLE
桌面应用程序框架技术

### DISCUSSION THREAD

**Discussion by wp2code** [wp2code/kaleido-client#2] [Q&A] [answered]
支持跨平台构建桌面应用程序的框架比较流行技术有什么

**Comment by W-Ping** [comment, marked as answer]
## 这些框架技术可以去了解一下
- 【electron】https://www.electronjs.org/
- 【QT】https://www.qt.io/
- 【flutter】https://flutter.dev/
- 【tauri】https://tauri.app/
